src/numbuf/thirdparty/download_thirdparty.sh

```diff
@@ -8,7 +8,7 @@ set -e
 TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
 if [ ! -d $TP_DIR/arrow ]; then
-  git clone https://github.com/pcmoritz/arrow.git "$TP_DIR/arrow"
+  git clone https://github.com/pcmoritz/arrow/ "$TP_DIR/arrow"
```

Comment:
Since Arrow-664 has been merged https://github.com/apache/arrow/commit/cd4544df89b60641f49bbb3104043c0ae07ef8a9, should we clone `https://github.com/apache/arrow.git`?